.github/workflows/checklinks.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Markdown links
+
+on:
+  schedule:
+  - cron: "0 13 * * 1"
```

Comment:
Probably best to run this on use trigger rather than the cron, so the user can get immediate feedback

Something like
```yaml
on:
   push 
```

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

Reply:
I think the idea is to check through all the links once a week hence the cron, but maybe a trigger on push would be useful as well.

Reply:
@eddiejaoude @EmmaDawsonDev 
Definitely a very good suggestion  👍🏻